modules/S05_firmware_details.sh

```diff
@@ -17,15 +17,25 @@
 #               Access:
 #                 firmware root path via $FIRMWARE_PATH
 #                 binary array via ${BINARIES[@]}
-
+export HTML_REPORT
```

Comment:
It's already exported in the emba.sh script. But I believe shellchecker doesn't like it, if you only set this variable and don't use it. But theres a hacky solution for this:
You can suppress the error message in this particular case, if you add a comment over `HTML_REPORT=1` with the right content: `# shellcheck disable=SCxxxx` - replace xxxx with the correct error code.

Reply:
But I want @m-1-k-3 opinion here.

Reply:
I would leave the export in here. Otherwise we have to include the disable comment on every usage of HTML_REPORT

Reply:
We leave it as it is. You can happily ignore this comment :D